packages/lib/server-only/document/send-document.tsx

```diff
@@ -11,9 +11,10 @@ import { DocumentStatus, SendStatus } from '@documenso/prisma/client';
 export type SendDocumentOptions = {
   documentId: number;
   userId: number;
+  resendEmails?: string[];
 };
 
-export const sendDocument = async ({ documentId, userId }: SendDocumentOptions) => {
+export const sendDocument = async ({ documentId, userId, resendEmails }: SendDocumentOptions) => {
```

Comment:
Can you avoid this and instead create a new router for resending a document?

You can also modify the subject of the emails that you send.